components/create_comment/create_comment.jsx

```diff
@@ -153,10 +157,10 @@ export default class CreateComment extends React.PureComponent {
         if (prevState.draft.uploadsInProgress.length < this.state.draft.uploadsInProgress.length) {
             this.scrollToBottom();
         }
+    }
 
-        if (prevProps.rootId !== this.props.rootId) {
-            this.focusTextbox();
-        }
+    onPostPinnedChange = () => {
+        this.focusTextbox();
```

Comment:
If we're just calling this function and nothing, it probably makes more sense to just pass this to the listener rather than make a new function